todo.js

```diff
@@ -7,17 +7,25 @@ const todoItemsList = document.querySelector('.todo-items')
 
 const totalTasks = document.querySelector(".total-tasks span")
 const completedTasks = document.querySelector(".completed-tasks span")
-const remainingTasks = document.querySelector(".remaining-tasks span")
+// const remainingTasks = document.querySelector(".remaining-tasks span")
 
 
+const currentDay=new Date()
```

Comment:
rename variable to currentDate or dateObject or something as it denotes the complete date